x/concentrated-liquidity/incentives_test.go

```diff
@@ -204,21 +203,6 @@ func expectedIncentivesFromUptimeGrowth(uptimeGrowths []sdk.DecCoins, positionSh
 	return totalRewards
 }
 
-// expectedForfeitureFromUptimeGrowth is similar to expectedIncentivesFromUptimeGrowth but returns a DecCoins type instead of a Coins type.
-// We cant reuse expectedIncentivesFromUptimeGrowth and normalize the result, since expectedIncentivesFromUptimeGrowth needs to normalize the totalRewards in each loop iteration.
-func expectedForfeitureFromUptimeGrowth(uptimeGrowths []sdk.DecCoins, positionShares sdk.Dec, timeInPool time.Duration, multiplier sdk.Int) sdk.DecCoins {
-	// Sum up rewards from all inputs
-	totalRewards := sdk.DecCoins(nil)
-	for uptimeIndex, uptimeGrowth := range uptimeGrowths {
-		if timeInPool >= types.SupportedUptimes[uptimeIndex] {
-			curRewards := uptimeGrowth.MulDecTruncate(positionShares).MulDecTruncate(multiplier.ToDec())
-			totalRewards = totalRewards.Add(curRewards...)
-		}
-	}
-
-	return totalRewards
-}
```

Comment:
Note to reviewer: method no longer needed since we no longer track forfeits with decCoins